### PR TITLE
Display Selected Filters on Filter Button

### LINKED ISSUE #7: feature: Display applied filters on filter button when tray is closed
the design of the filters should somewhat resemble the pill-buttons in the filter tray. Although they should not be clickable

### PR DESCRIPTION
**Pull request by scarlettsteph07**
# What does this PR do?
Fixes #7 
Displays the selected filters on filter button

# Demo
* Before:
image

* After:
image

### CODE REVIEW

**Comment by scarlettsteph07** [comment]
@deepredsky, thanks for your review! The comments were addressed on https://github.com/jobn/braid/pull/9/commits/8dd2b354d49170eff65fe947839e83fdbc0d6c1e

File: src/Filters.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ const Filters = ({
       </div>
       <div className="media-content">
         <div className="field is-grouped is-grouped-multiline">
-          {uniqueOwnerIds.map(id => (
-            <p className="control">
+          {uniqueOwnerIds.map((id, index) => (
```
Comment (scarlettsteph07) [inline src/Filters.js]: https://github.com/jobn/braid/pull/9/commits/cf4cc572898b6354065f81081bb903b246d1fc28 gets rid of this warning: 

screen shot 2018-10-21 at 11 51 18

Comment (scarlettsteph07) [inline src/Filters.js]: https://github.com/jobn/braid/pull/9/commits/cf4cc572898b6354065f81081bb903b246d1fc28 gets rid of this warning:

screen shot 2018-10-21 at 11 51 18

Comment (deepredsky) [reply src/Filters.js]: Thanks for fixing the warning!

while this change does fix the warning, using array index as key is known to have issues and considered an anti-pattern. Why not just use the `id` as key here?

File: src/Project.js [hunk 2]
```diff
@@ -152,7 +152,7 @@ class Project extends Component {
         </section>
 
         <Footer>
-          <Tray title="Filters">
+          <Tray selectedOwners={selectedOwners} people={people} selectedTypes={selectedTypes} title="Filters">
```
Comment (jobn) [inline src/Project.js]: as the `Tray` is a general component I think it should not know anything about filters, selectedOwners and such. Going forward we would add another tray that has some settings about how the stories should be displayed. And that tray should not know anything about `selectedTypes` or the other filter props.

I think a better way would be to implement the Tray label using a render prop, something like this:
```
<Tray renderLabel={() => (
  <Fragment>
    <span>Filters</span>
    {selectedOwners.length > 0 && <Owners ownerIds={selectedOwners} />}
    { /* something similar for selectedTypes */ }
  </Fragment>
)}>
  <Filters
    uniqueOwnerIds={uniqueOwnerIds}
   ...
</Tray>
```

here is the react documentation on render props: https://reactjs.org/docs/render-props.html

**Pull request closed without merging**